core/src/messages/agent.rs

```diff
@@ -107,6 +107,7 @@ impl JetStreamable for DroneLogMessage {
 
 #[derive(Serialize, Deserialize, Debug)]
 pub struct BackendStatsMessage {
+    cluster: Option<ClusterName>,
```

Comment:
We'll change this to a required field once all the drones are updated?

Reply:
Exactly. This just allows us to push a version of the controller that is able to accept messages that don't include it.